bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -842,15 +848,18 @@ public int runCmd(CommandLine cmdLine) throws Exception {
                 return -1;
             }
 
-            ZooKeeper zk = null;
             LedgerManagerFactory mFactory = null;
             LedgerManager m = null;
             try {
-                zk = ZooKeeperClient.newBuilder()
-                        .connectString(bkConf.getZkServers())
-                        .sessionTimeoutMs(bkConf.getZkTimeout())
-                        .build();
-                mFactory = LedgerManagerFactory.newLedgerManagerFactory(bkConf, zk);
+                mFactory = LedgerManagerFactory.newLedgerManagerFactory(
+                    bkConf,
+                    new ZkLayoutManager(
```

Comment:
try to avoid using ZkLayoutManager directly

Reply:
Thanks, will change it.